modules/ppcp-compat/src/Settings/StylingSettingsMapHelper.php

```diff
@@ -70,6 +71,9 @@ public function mapped_value( string $old_key, array $styling_models ) {
 			case 'smart_button_locations':
 				return $this->mapped_smart_button_locations_value( $styling_models );
 
+			case 'smart_button_enable_styling_per_location':
+				return true;
```

Comment:
Does it make sense to add a filter here to allow programmatically disabling this to use a single styling option for all locations?

Reply:
@stracker-phil I think it doesn't make sense for the new settings, as it doesn't have any option for using a single styling